be2-scala/src/main/scala/ch/epfl/pop/pubsub/MessageRegistry.scala

```diff
@@ -13,13 +13,13 @@ import ch.epfl.pop.model.network.method.message.data.{ActionType, MessageData, O
 import ch.epfl.pop.pubsub.MessageRegistry.RegisterEntry
 import ch.epfl.pop.pubsub.graph.handlers._
 import ch.epfl.pop.pubsub.graph.validators._
-import ch.epfl.pop.pubsub.graph.{GraphMessage, JsonString, SchemaValidator}
+import ch.epfl.pop.pubsub.graph.{GraphMessage, JsonString, SchemaVerifier}
```

Comment:
we could even import SchemaVerifier._, in order to be able to remove it from the createSchemaVerifier function calls at lines 58-88 (optional)